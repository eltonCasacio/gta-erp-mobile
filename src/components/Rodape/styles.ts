import {Dimensions} from 'react-native';
import styled from 'styled-components/native';

type WrapperProps = {
  Dimension: Dimensions;
};

export const Wrapper = styled.View<WrapperProps>`
  flex-direction: row;
  justify-content: space-around;
  height: ${Dimensions.get('window').height / 5};
`;

export const WrapperMenu = styled.View`
  width: ${Dimensions.get('window').width / 4.2};
`;
