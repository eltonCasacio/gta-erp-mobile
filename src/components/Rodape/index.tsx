import React from 'react';
import {Wrapper} from './styles';

import Menu from '../Menu';

type RodapeProps = {
  navigation: any;
};

const Rodape = ({navigation}: RodapeProps) => {
  return (
    <Wrapper>
      <Menu navigation={navigation} />
    </Wrapper>
  );
};

export default Rodape;
