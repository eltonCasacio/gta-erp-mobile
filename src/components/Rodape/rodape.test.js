import React from 'react';
import renderer from 'react-test-renderer';

import Rodape from '.';

describe('<Rodape />', () => {
  it('should render default layout', () => {
    const tree = renderer.create(<Rodape />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
