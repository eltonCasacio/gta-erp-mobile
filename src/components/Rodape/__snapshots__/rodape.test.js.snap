// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rodape /> should render default layout 1`] = `
<View
  Dimension={[Function]}
  style={
    Array [
      Object {
        "flexDirection": "row",
        "height": 100,
        "justifyContent": "space-around",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "width": 80,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#5762F2",
          "height": "100%",
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fafafa",
              "fontSize": 12,
              "textAlign": "center",
            },
          ]
        }
      >
        REGISTRAR HORÁRIO
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "width": 80,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#5762F2",
          "height": "100%",
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fafafa",
              "fontSize": 12,
              "textAlign": "center",
            },
          ]
        }
      >
        DOWNLOAD DE HOLERITE
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "width": 80,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#5762F2",
          "height": "100%",
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fafafa",
              "fontSize": 12,
              "textAlign": "center",
            },
          ]
        }
      >
        RELATÓRIO DE MARCAÇÃO
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "width": 80,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#5762F2",
          "height": "100%",
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 5,
          "paddingLeft": 5,
          "paddingRight": 5,
          "paddingTop": 5,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fafafa",
              "fontSize": 12,
              "textAlign": "center",
            },
          ]
        }
      >
        INFORMAÇÕES DA EMPRESA
      </Text>
    </View>
  </View>
</View>
`;
